### ISSUE TITLE
set_automated_followed_pool error 'datetime.datetime' object has no attribute 'timestamp'

### ISSUE THREAD

**Issue by riccardorighettiphoto** [InstaPy/InstaPy#1495]
Hello,
I am running InstaPy without any issues so far, but at the top of log the error in the subject comes up.
What kind of error is it?

**Comment by uluQulu** [comment]
Hi @riccardorighettiphoto 
Well I think you're using **old version** of my repo (InstaPy-with-QS) or my commit for `unfollow_after` parameter

The problem you encountered is **specific to Python 2** which does not support _datetime to seconds_ conversion with `timestamp()` _AND_ **some time ago** I have added support **for** Python 2 by **using** `total_seconds()` method

Please, **pull** the latest commits _OR_ redownload that repository _OR_ apply new changes in there manually (`set_automated_followed_pool()` function in `unfollow_util.py`)

**Comment by riccardorighettiphoto** [comment]
Hello @uluQulu,
Thanks for the hint. I did not pull your InstaPy with QS repository, but I am on Python 2.7 indeed, so I amended my ‘unfollow_util.py’ file manually, with the function in your file. It looks like working now, I don’t have anymore that error. I will check if it unfollows users in the following days because I have setup a one-week period before to instruct an unfollow task. I’ll report again if any issue comes up. Thanks for now.
